### PR TITLE
Fix em

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1-Moved EM test cases on python3 
2-Rename cpupower to em_cpupower 
3-Fix issue while fetching Platform details 

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

File: cpu/em_cpupower.py [hunk 1]
```diff
@@ -19,10 +19,11 @@
 from avocado import Test
 from avocado import main
 from avocado import skipIf
-from avocado.utils import process, distro, cpu
+from avocado.utils import process, distro
 from avocado.utils.software_manager import SoftwareManager
 
-IS_POWER_NV = 'PowerNV' in cpu._get_cpu_info()
+# TODO : Logic need to change when we have lib fix
+IS_POWER_NV = 'PowerNV' in open('/proc/cpuinfo', 'r').read()
```
Comment (narasimhan-v) [inline cpu/em_cpupower.py]: @PraveenPenguin can't we wait for lib fix ?

File: cpu/em_cpuidle.py [hunk 2]
```diff
@@ -87,8 +87,9 @@ def read_from_device_tree(self):
         os.chdir('/proc/device-tree/ibm,opal/power-mgt')
         cmd_args = ['lsprop', 'ibm,cpu-idle-state-names']
         output_string = subprocess.Popen(
-            cmd_args, stdout=subprocess.PIPE).communicate()[0]
+            cmd_args, stdout=subprocess.PIPE).communicate()[0].decode("utf-8")
         output = re.findall('\"[a-zA-Z0-9_]+\"', output_string)
+
```
Comment (narasimhan-v) [inline cpu/em_cpuidle.py]: empty line needed ?

File: cpu/em_cpupower.py (hunk 1 above)
Comment (PraveenPenguin) [reply cpu/em_cpupower.py]: @narasimhan-v not sure how much that will take as my ealier PR also i need to rework , i think we can have this in and later in on time we can change all applicable test in one commit

File: cpu/em_cpuidle.py (hunk 2 above)
Comment (PraveenPenguin) [reply cpu/em_cpuidle.py]: will remove this , will include this in last commit as  i need to rebase

**Comment by PraveenPenguin** [comment]
@narasimhan-v pushed the changes please have a look , adopting lib will take time as still old lib patch need to be in and after lib fix will  adopt on all the test cases

**Pull request merged**